src/pages/_app.js

```diff
@@ -69,4 +42,4 @@ class MyApp extends App {
   }
 }
```

Comment:
I think removing all this logic from the app is a good idea.
Also the `_app`’s context signature is different from others??? That’s why I updated every `context.ctx. …` to `context. …`.